mevshare/builders.go

```diff
@@ -0,0 +1,238 @@
+package mevshare
+
+import (
+	"context"
+	"errors"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ybbus/jsonrpc/v3"
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+)
+
+type BuilderAPI uint8
+
+const (
+	BuilderAPIRefRecipient BuilderAPI = iota
```

Comment:
nitpick (non-blocking): since this is a constant, the full word may help be more clear, i.e. 
```suggestion
	BuilderAPIRefundRecipient BuilderAPI = iota
```

Reply:
sure, why not